composer/plan.sh

```diff
@@ -0,0 +1,29 @@
+pkg_name=composer
+pkg_distname=$pkg_name
```

Comment:
It looks like in all of these examples `pkg_distname` is completely useless. I thought I had seen it used where the name of a package was different from the directory it used to unpack things but I don't see that here, this is all cargo-culting or just old plans.